module/VuFind/sql/pgsql.sql

```diff
@@ -189,6 +189,22 @@ UNIQUE (session_id)
 CREATE INDEX last_used_idx on session(last_used);
 
 --
+-- Table structure for table external_session
+--
+
+DROP TABLE IF EXISTS "external_session";
+
+CREATE TABLE external_session (
+id SERIAL,
+session_id varchar(128),
```

Comment:
Copy-paste from session table without cross-checking. Now fixed. The same inconsistency is there in the session table, but I didn't touch it since it's out of scope here. I'm also unable to verify that the Postgresql stuff actually works.